### PR TITLE
Update the XML document format of 5 committed wiki pages

### PR DESCRIPTION
**Pull request by SrijanJha28**

### CODE REVIEW

File: xwiki-platform-core/xwiki-platform-sandbox/src/main/resources/Sandbox/TestPage1.de.xml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="UTF-8"?>
+<?xml version='1.1' encoding='UTF-8'?>
```
Comment (vmassol) [inline xwiki-platform-core/xwiki-platform-sandbox/src/main/resources/Sandbox/TestPage1.de.xml]: I don't understand the reason for this change could you explain?

File: xwiki-platform-core/xwiki-platform-sandbox/src/main/resources/Sandbox/TestPage1.de.xml [hunk 2]
```diff
@@ -20,31 +20,28 @@
...
-  <validationScript/>
-  <comment/>
+  <comment>Install extension [org.xwiki.platform:xwiki-platform-sandbox/9.11]</comment>
   <minorEdit>false</minorEdit>
   <syntaxId>xwiki/2.1</syntaxId>
   <hidden>false</hidden>
   <content>Klicken Sie auf **"Bearbeiten"**, um diese Seite zu editieren. Wenn Sie fertig mit dem Bearbeiten sind, klicken Sie **"Speichern &amp; Ansehen"**, um sich die Ergebnisse anzusehen!
 
-= Hier ist ein wenig Fülltext, damit Sie sich vorstellen können, wie eine Seite aussieht =
+= Hier ist ein wenig FÃ¼lltext, damit Sie sich vorstellen kÃ¶nnen, wie eine Seite aussieht =
```
Comment (vmassol) [inline xwiki-platform-core/xwiki-platform-sandbox/src/main/resources/Sandbox/TestPage1.de.xml]: Seems you broke the translations :(

File: xwiki-platform-core/xwiki-platform-sandbox/src/main/resources/Sandbox/TestPage1.de.xml [hunk 3]
```diff
@@ -20,31 +20,28 @@
...
+  <author>XWiki.superadmin</author>
+  <contentAuthor>XWiki.superadmin</contentAuthor>
+  <date>1513606718000</date>
+  <contentUpdateDate>1513606718000</contentUpdateDate>
   <version>1.1</version>
   <title>Sandbox Test Page 1</title>
-  <defaultTemplate/>
-  <validationScript/>
-  <comment/>
+  <comment>Install extension [org.xwiki.platform:xwiki-platform-sandbox/9.11]</comment>
```
Comment (vmassol) [inline xwiki-platform-core/xwiki-platform-sandbox/src/main/resources/Sandbox/TestPage1.de.xml]: The comment should be empty. Did you run `mvn xar:format` and `mvn xar:verify`?

File: xwiki-platform-core/xwiki-platform-sandbox/src/main/resources/Sandbox/TestPage1.fr.xml [hunk 4]
```diff
@@ -20,29 +20,26 @@
...
-  <defaultTemplate/>
-  <validationScript/>
-  <comment/>
+  <title>Page de Test 1 du bac Ã  sable</title>
+  <comment>Install extension [org.xwiki.platform:xwiki-platform-sandbox/9.11]</comment>
   <minorEdit>false</minorEdit>
   <syntaxId>xwiki/2.1</syntaxId>
   <hidden>false</hidden>
-  <content>Cliquez sur **"Modifier"** puis altérez le contenu de la page enfin cliquez sur **"Enregistrer &amp; Fermer** pour visualiser le résultat de vos modifications dans la page !
+  <content>Cliquez sur **"Modifier"** puis altÃ©rez le contenu de la page enfin cliquez sur **"Enregistrer &amp; Fermer** pour visualiser le rÃ©sultat de vos modifications dans la page !
```
Comment (tmortagne) [inline xwiki-platform-core/xwiki-platform-sandbox/src/main/resources/Sandbox/TestPage1.fr.xml]: You seems to have some encoding issue somewhere.

File: xwiki-platform-core/xwiki-platform-sandbox/src/main/resources/Sandbox/TestPage1.lv.xml [hunk 5]
```diff
@@ -20,31 +20,27 @@
...
-  <defaultTemplate/>
-  <validationScript/>
   <comment/>
   <minorEdit>false</minorEdit>
   <syntaxId>xwiki/2.1</syntaxId>
   <hidden>false</hidden>
   <content>Klikšķiniet uz **"Mainīt"** saites, lai mainīt šīs lapas saturu. Pēc tam klikšķiniet uz  **"Saglabāt un skatīt"**, lai apskatītu kā izskatās jūs mainītais lapas saturs.
 
 = Šeit ir parauga teksts, lai jūs redzētu, kā lapa izskatās =
-
```
Comment (tmortagne) [inline xwiki-platform-core/xwiki-platform-sandbox/src/main/resources/Sandbox/TestPage1.lv.xml]: This looks wrong, there should not be any modification in the content.

**Comment by SrijanJha28** [comment]
Done :)

File: xwiki-platform-core/xwiki-platform-sandbox/src/main/resources/Sandbox/TestPage1.lv.xml [hunk 6]
```diff
@@ -46,5 +43,5 @@
 
 = Šeit ir parauga teksts, lai jūs redzētu, kā lapa izskatās =
 
-Arvien vēl pūta dienvidus vakara vējš, un arvien vēl milzīgais ledus gabals peldēja tālāk un tālāk jūrā iekšā. Uz ledus gabala atradās četrpadsmit zvejnieki un divi zirgi. Ļaudis bij nodarbojušies ar āliņģu ciršanu un tīklu laišanu ūdenī, un neviens no viņiem nebij pamanījis, ka ledus no jūrmalas bij sācis virzīties projām.</content>
+   Arvien vēl pūta dienvidus vakara vējš, un arvien vēl milzīgais ledus gabals peldēja tālāk un tālāk jūrā iekšā. Uz ledus gabala atradās četrpadsmit zvejnieki un divi zirgi. Ļaudis bij nodarbojušies ar āliņģu ciršanu un tīklu laišanu ūdenī, un neviens no viņiem nebij pamanījis, ka ledus no jūrmalas bij sācis virzīties projām.</content>
```
Comment (tmortagne) [inline xwiki-platform-core/xwiki-platform-sandbox/src/main/resources/Sandbox/TestPage1.lv.xml]: Content modified here.

**Comment by SrijanJha28** [comment]
Done :)

**Comment by tmortagne** [comment]
The content is still modified.

**Comment by SrijanJha28** [comment]
Can you please quote the lines where it is modified

**Comment by tmortagne** [comment]
All good now, thanks for your contribution !

**Comment by tmortagne** [comment]
@SrijanJha28 actually you forgot to execute mvn xar:format in your last version...

**Pull request merged**